tests/unit/test_grammar.py

```diff
@@ -2,6 +2,12 @@
 from guidance import gen, models, optional, select
 
 
+def test_readable_output():
```

Comment:
I just wanted to make any test for my object not breaking anything, I guess someone with more experience on writing tests could do much better